tests/model_serving/model_runtime/triton/basic_model_deployment/utils.py

```
@@ -0,0 +1,207 @@
+) -> None:
+    response = run_triton_inference(
+        pod_name=pod_name,
+        isvc=isvc,
+        input_data=input_query,
+        model_version=model_version,
+        protocol=protocol,
+        root_dir=root_dir,
+    )
+    assert response == response_snapshot, f"Output mismatch: {response} != {response_snapshot}"
+
+from tests.model_serving.model_runtime.triton.constant import ACCELERATOR_IDENTIFIER, TEMPLATE_MAP
```

Ruff (0.11.9)

Module level import not at top of file

(E402)